dracut/anaconda-copy-prefixdefname.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Copy over persistent network device names to anaconda environment
+
+CMDLINE=$(cat /proc/cmdline)
+if [[ ${CMDLINE} =~ "net.ifnames.prefix=" ]]; then
+  mkdir -p /run/initramfs/state/etc/systemd/network
+  cp /etc/systemd/network/71-net-ifnames-prefix* /run/initramfs/state/etc/systemd/network
```

Comment:
Is the `71` prefix fixed number? Couldn't this change in the future for some reason?

Reply:
I discussed this with Michal Sekletar and he recommended this approach (eg rather than copying content of /etc/systemd/network which at this moment contains only the prefixdevname configuation files).